src/modelio.cs

```diff
@@ -340,6 +345,11 @@ interface MDLLight
 		[Export ("lightType")]
 		MDLLightType LightType { get; set; }
 
+		// Added in iOS 10 SDK but it is supposed to be present in iOS 9.
+		[Mac (10,12)]
+		[Export ("colorSpace")]
+		string ColorSpace { get; set; }
```

Comment:
@spouliot there is 0 information about it, in neither the header nor the online doc ):

Do you want me to make it an `NSString`?